app/assets/stylesheets/reservations/_new.css.scss

```diff
@@ -1,3 +1,13 @@
 .warning-icon {
   color: $btn-warning-bg;
 }
+
+.edit_reservation_form {
```

Comment:
I believe standard practice is to use dashes in CSS selectors, not underscores (e.g., `edit-reservation-form` instead of `edit_reservation_form`). Can you double check that and, if it's true, update your styling (here and in the views)? Thanks!